.gitmodules

```diff
@@ -1,4 +1,3 @@
-[submodule "src/core_atmosphere/physics/GSL_cloud_physics"]
-	path = src/core_atmosphere/physics/GSL_cloud_physics
-	url = https://github.com/AndersJensen-NOAA/GSL_cloud_physics.git
-	branch = develop
\ No newline at end of file
+[submodule "src/core_atmosphere/physics/TEMPO"]
+	path = src/core_atmosphere/physics/TEMPO
+	url = https://github.com/NCAR/TEMPO.git
```

Comment:
@AndersJensen-NOAA Anders, why are changing GSL_cloud_physics to TEMPO? It seems to me less descriptive about its content and about the developer.

Reply:
@tanyasmirnova EMC wants to collaborate on microphysics development, so (after lots of discussion) we decided to move the submodule to an NCAR (neutral) repository and not keep it GSL specific.

Reply:
@AndersJensen-NOAA Thank you for explaining.